src/one_dragon/base/operation/application/application_config.py

```
@@ -0,0 +1,18 @@
+class ApplicationConfig(YamlOperator):
+
+    def __init__(self, app_id: str, instance_idx: int, group_id: str):
+        f"""
+        应用配置，保存在 config/{instance_idx}/{group_id}/{app_id}.yml 文件中
+        默认 group_id='one_dragon'
+        
+        Args:
+            app_id: 应用ID
+            instance_idx: 实例下标
+            group_id: 分组ID
+        """
```

Ruff (0.12.2)

f-string used as docstring. Python will interpret this as a joined string, rather than a docstring.

(B021)